src/topology/locally_constant/algebra.lean

```diff
@@ -110,4 +117,29 @@ instance [ring Y] : ring (locally_constant X Y) :=
 instance [comm_ring Y] : comm_ring (locally_constant X Y) :=
 { .. locally_constant.comm_semiring, .. locally_constant.ring }
 
+variables (R : Type*)
+
+instance [has_scalar R Y] : has_scalar R (locally_constant X Y) :=
+{ smul := λ r f,
+  { to_fun := r • f,
+    is_locally_constant := ((is_locally_constant f).comp ((•) r) : _), } }
+
+@[simp] lemma coe_smul [has_scalar R Y] (r : R) (f : locally_constant X Y) :
+  ⇑(r • f) = r • (f : X → Y) :=
```

Comment:
I think you can actually just write this as:
```suggestion
  ⇑(r • f) = r • f :=
```
but no need to change it unless you prefer that spelling.

Reply:
It gets the line to 100 characters to I'll take it!